src/# _*_ encoding: utf-8 _*_.py

```diff
@@ -0,0 +1,157 @@
+# _*_ encoding: utf-8 _*_
+"""Python graph."""
+
+from deque import Deque
+from stack import Stack
+from queue import Queue
+
+
+class Graph(object):
+    """Class Graph."""
+
+
+def __init__(self):
+    """Initiatiliztion graph."""
+    self.graph = {}
+
+
+def node(self):
+    """Nodes in graph."""
+    node = []
+    for key in self.graph:
+        node.append(key)
+    return node
+
+
+def edges(self):
+    """List of edges in graph."""
+    return [{(key1, key2): self.graph[key1][key2]} for key1 in self.graph for key2 in self.graph[key1]]
+
+
+def add_node(self, val):
+    """Add node to graph."""
+    if val in self.graph:
+        pass
+    else:
+        self.graph[val] = {}
+
+
+def add_edge(self, val, val2):
+    """Add edge to graph."""
+    if val not in self.graph:
+        self.add_node(val)
+    if val2 not in self.graph:
+        self.add_node(val2)
+    if val2 in self.graph[val]:
+        pass
+    else:
+        self.graph[val].append(val2)
```

Comment:
The nodes in your graph aren't pointing at lists for each. They're pointing at dictionaries, and there's no `append` method on a dictionary.